### PR TITLE
Fixes #275 Working with embedded scenes

### LINKED ISSUE #275: The inspector does not work with embedded scenes
Load the wikipedia example. This is what I see:

wilkipedia

### PR DESCRIPTION
**Pull request by fernandojsg**
With the help of some code stolen from https://github.com/aframevr/aframe-inspector/pull/294 by  @ngokevin ❤️ 

wikipedia

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -736,6 +743,10 @@ a.help-link:hover {
 
 /* This is a temporaly hack, we should style the editor instead of overwriting
    the a-scene to fix the "display: block" issue. */
-a-scene {
+.aframe-inspector-opened a-scene {
```
Comment (ngokevin) [inline src/css/main.css:751]: What about multiple scenes? Maybe we should keep track of the actively inspected scene.

**Comment by ngokevin** [comment]
Got some peeking DOM elements. What if we just `display: none !important` everything that is not the canvas?

screen shot 2016-08-17 at 6 02 28 pm

File: src/css/main.css (hunk 1 above)
Comment (fernandojsg) [inline src/css/main.css:751]: We should keep track of that yep, but right now there's no support for multiple scenes on the rest of the code as we're looking for example for `a-scene` in query selector. So we should do another PR to achieve that, and also expose the other scenes on the inspector UI probably so you could switch them.

**Comment by ngokevin** [comment]
Would be good to hide every element on the page:

screen shot 2016-08-17 at 6 03 16 pm

File: src/css/main.css (hunk 1 above)
Comment (ngokevin) [inline src/css/main.css:751]: ok np

**Comment by fernandojsg** [comment]
@ngokevin umm what about a background-color for `canvas` ?

**Comment by fernandojsg** [comment]
and `z-index` for canvas and UI ?

**Comment by ngokevin** [comment]
Yeah, could work if we save/restore the background color and add a high-enough z-index.

**Comment by fernandojsg** [comment]
I've added `background-color` to `canvas` and `z-index` to `canvas` and `panels` but I can't get rid of some elements in the top of the canvas :| Any idea @ngokevin ?
screenshot 2016-08-18 03 26 23
screenshot 2016-08-18 03 26 14

**Comment by ngokevin** [comment]
r+
